ztp/kube-compare-reference/Makefile

```diff
@@ -24,4 +27,25 @@ metadata_lint: kubectl-cluster_compare
 
 .PHONY: clean
 clean:
-	rm -rf kubectl-cluster_compare
+	rm -rf kubectl-cluster_compare Chartv1 renderedv1 /usr/local/bin/helm
```

Comment:
This seems like it would only work if run as root?

We shouldn't be adding or removing tools in /usr/local/bin as part of our build process.